prime-dotnet-webapi/Migrations/20220316172231_AddNewPharmacyTechnicianToa.cs

```diff
@@ -0,0 +1,38 @@
 provisions were not included.\r\n        </p>\r\n\r\n      </li>\r\n      <li>\r\n\r\n        <p>\r\n          <strong>Survival.</strong> Any provision of this Agreement that expressly or by its nature\r\n          continues after termination, shall survive termination of this Agreement.\r\n        </p>\r\n\r\n      </li>\r\n      <li>\r\n\r\n        <p>\r\n          <strong>Governing Law.</strong> This Agreement will be governed by and will be construed and\r\n          interpreted in accordance with the laws of British Columbia and the laws of\r\n          Canada applicable therein.\r\n        </p>\r\n\r\n      </li>\r\n      <li>\r\n\r\n        <p>\r\n          <strong>Assignment Restricted.</strong> Your rights and obligations under this Agreement may\r\n          not be assigned without the prior written approval of the Province.\r\n        </p>\r\n\r\n      </li>\r\n      <li>\r\n\r\n        <p>\r\n          <strong>Waiver.</strong> The failure of the Province at any time to insist on performance of any\r\n          provision of this Agreement by you is not a waiver of its right subsequently to\r\n          insist on performance of that or any other provision of this Agreement.\r\n        </p>\r\n\r\n      </li>\r\n      <li>\r\n\r\n        <p>\r\n          <strong>Province May Modify this Agreement.</strong> The Province may amend this\r\n          Agreement, including this section, at any time in its sole discretion:\r\n        </p>\r\n\r\n        <ol type=\"i\">\r\n          <li>\r\n            by written notice to you, in which case the amendment will become\r\n            effective upon the later of (A) the date notice of the amendment is first\r\n            delivered to you, or (B) the effective date of the amendment specified by\r\n            the Province, if any; or\r\n          </li>\r\n          <li>\r\n            by publishing notice of any such amendment in the PharmaCare\r\n            Newsletter, in which case the notice will specify the effective date of the\r\n            amendment, which date will be at least 30 (thirty) days after the date that\r\n            the PharmaCare Newsletter containing the notice is first published.\r\n          </li>\r\n        </ol>\r\n\r\n        <p>\r\n          If you use PharmaNet after the effective date of an amendment described in (i) or\r\n          (ii) above, you will be deemed to have accepted the corresponding amendment,\r\n          and this Agreement will be deemed to have been so amended as of the effective\r\n          date. If you do not agree with any amendment for which notice has been provided\r\n          by the Province in accordance with (i) or (ii) above, you must promptly (and in\r\n          any event before the effective date) cease all access or use of PharmaNet by yourself and take the steps\r\n          necessary to terminate this Agreement in accordance\r\n          with section 10.\r\n        </p>\r\n\r\n      </li>\r\n    </ol>\r\n\r\n  </li>\r\n</ol>\r\n");
```

Comment:
I am wondering why in the generated migration there is an update to the previous Pharm Tech TOA?

Reply:
Good point, looking into it

Reply:
You might have to do these steps to rollback and re-generate your migration:
1. `dotnet ef database update <migration prior to your changes> --verbose` (https://docs.microsoft.com/en-us/ef/core/cli/dotnet#dotnet-ef-database-update)
2. Overwrite `ApiDbContextModelSnapshot.cs` with the content of version prior to your changes
3. Delete `prime-dotnet-webapi/Migrations/20220316172231_AddNewPharmacyTechnicianToa.cs`
4. Run `dotnet ef migrations add <name of your migration>`

Then you have to commit the deletion of `prime-dotnet-webapi/Migrations/20220316172231_AddNewPharmacyTechnicianToa.cs` so it doesn't show up in your PR.

Reply:
Fixed